js/index.js

```diff
@@ -39,4 +39,63 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+//IMAGES HEADER LOGO
+let headerLogo = document.getElementById("cta-img");
+headerLogo.setAttribute('src', siteContent["cta"]["img-src"]);
+//MIDDLE IMAGE
+let midImg = document.getElementById("middle-img");
+midImg.setAttribute('src', siteContent["main-content"]["middle-img-src"]);
+
+
+//NAV
+const anchorItems = document.querySelectorAll('a');
+
+const anchText = document.querySelectorAll("nav a");
+for (let i = 0; i < anchText.length; i++) {
+  anchText[i].textContent = siteContent["nav"][`nav-item-${i + 1}`];
+  anchText[i].style.color = "crimson";
+}
+
+
+//HEADER TEXT
+const cta = document.querySelector('.cta-text h1');
+cta.textContent = siteContent["cta"]["h1"];
+
+
+//BUTTON
+const btn = document.querySelector("button");
+btn.textContent = siteContent["cta"]["button"];
+
+
+//TOP CONTENT
+//FIRST H4
+const topContent = document.querySelector('.text-content h4');
```

Comment:
I would just query for all the h4s here and then change them based on their order(index), same with the Ps